ddtrace/internal/_encoding.pyx

```diff
@@ -244,6 +244,9 @@ cdef class MsgpackStringTable(StringTable):
     cdef insert(self, object string):
         cdef int ret
 
+        if len(string) > self.max_size:
+            string = "<dropped in ddtrace.internal._encoding.pyx due to buffer overrun>"
```

Comment:
I think we should include the size of the buffer and the size of the string that we attempted to include - this could assist in debugging issues.